Makefile

```diff
@@ -95,44 +95,44 @@ game.o: /usr/include/bits/waitstatus.h /usr/include/bits/floatn.h
 game.o: /usr/include/bits/floatn-common.h /usr/include/sys/types.h
```

Comment:
This goo is silly. We need to find a way not to do this.

Reply:
I think you have addressed this in your new Makefile